solana/program_v2/packages/tests/src/pass-suite/expire.test.ts

```diff
@@ -72,27 +80,109 @@ describe('Expire a pass', () => {
   });
 
   it('Expires a pass', async () => {
-    await gatekeeperService.expirePass(passAccount).rpc();
+    await gatekeeperService
+      .expirePass(
+        passAccount,
+        TOKEN_PROGRAM_ID,
+        mint,
+        gatekeeperAta.address,
+        networkAta.address,
+        funderAta.address
+      )
+      .rpc();
+
     const updatedPass = await gatekeeperService.getPassAccount(
       subject.publicKey
     );
 
+    const funderAtaAccountInfo = await gatekeeperService
+      .getConnection()
+      .getAccountInfo(funderAta.address);
+
+    const networkAtaAccountInfo = await gatekeeperService
+      .getConnection()
+      .getAccountInfo(networkAta.address);
+
+    const gatekeeperAtaAccountInfo = await gatekeeperService
+      .getConnection()
+      .getAccountInfo(gatekeeperAta.address);
+
+    const funderAccount = AccountLayout.decode(funderAtaAccountInfo!.data);
+    const networkAccount = AccountLayout.decode(networkAtaAccountInfo!.data);
+    const gatekeeperAccount = AccountLayout.decode(
+      gatekeeperAtaAccountInfo!.data
+    );
+
+    // Assert
+    expect(funderAccount.amount).to.equal(0n);
+    expect(networkAccount.amount).to.equal(2n);
+    expect(gatekeeperAccount.amount).to.equal(1998n);
+
```

Comment:
@JulianSpring do these numbers seem correct? (keep in mind that some of the funds were transfered during issue already

Reply:
Yeah think so.
Funder accounts starts with 2000
Cost of issue is 1000 and split is 0.1%
So - gatekeeper gets 999 and network 1
Cost of refresh is 1000
so - gatekeeper gets 999 ->`1998` and network gets 1 -> `2`